Phases/Assessments/Phase_3.md

```diff
@@ -0,0 +1,35 @@
+# Phase 3 Assessment
+
+The phase 3 assessment will test your ability to combine all of your foundational skills to build real, full-stack web applications both alone and in small teams.
+
+Note that the phase 3 assessment differs significantly from the previous assessments in that you will be **pair programming as part of your evaluation interview**. A key skill to learn in phase 3 is how to build code in team, and the interviewer will be assessing your ability to work with them to build out a feature.
+
+## Skills Assessed
+
+- HTTP Applications (Intermediate Level)
+- HTML & CSS (Intermediate Level)
+- HTTP (Intermediate Level)
+- SQL (Intermediate Level)
+- Programming (Intermediate Level)
+
+Some examples of specific things you _may_ be required to do as part of the assessment (this is **not** an exhaustive list):
+
+- Build full-stack, database-backed web application with Node.js + Express, Postgres, and EJS
+- Implement user authentication (sign in with username + password)
```

Comment:
we should clarify that we mean encrypted cookie based auth and not http basic auth